src/documents/lang.md

```diff
@@ -0,0 +1,140 @@
+        "container": ".gpiic-fd-prefsEditor-panel-lang",
+        "template": "%templatePrefix/lang.html",
+        "message": "%messagePrefix/lang.json",
+        "stringArrayIndex": {
+            "lang": ["lang-en-US", "lang-fr-FR", "lang-es-MX", "lang-de-DE", "lang-nl-NL", "lang-sv-SE"],
+            "tooltip": ["lang-en-US-tooltip", "lang-fr-FR-tooltip", "lang-es-MX-tooltip", "lang-de-DE-tooltip", "lang-nl-NL-tooltip", "lang-sv-SE-tooltip"],
+            "tooltipAtSelect": ["lang-en-US-tooltipAtSelect", "lang-fr-FR-tooltipAtSelect", "lang-es-MX-tooltipAtSelect", "lang-de-DE-tooltipAtSelect", "lang-nl-NL-tooltipAtSelect", "lang-sv-SE-tooltipAtSelect"]
+        }
+    }
+}
+```
+
+Working in conjunction with the Auxiliary Schema, the type, its default value and range of
+the language preference are defined in the
+[Primary Schema](http://docs.fluidproject.org/infusion/development/PrimarySchemaForPreferencesFramework.html):
+```javascript
+fluid.defaults("gpii.firstDiscovery.schemas.language", {
+    gradeNames: ["fluid.prefs.schemas"],
+    schema: {
+        "gpii.firstDiscovery.language": {
+            "type": "string",
+            "default": "en-US",
+            "enum": ["en-US", "fr-FR", "es-MX", "de-DE", "nl-NL", "sv-SE"]
+        }
+    }
+});
+```
+
+*Option 2*: Outside the context of the First Discovery Tool, developers may wish to create a standalone component:
+```javascript
+var myPanel = gpii.firstDiscovery.panel.lang(container, options);
+```
+
+## Grades
+
+This component uses the following base
+[grades](http://docs.fluidproject.org/infusion/development/ComponentGrades.html):
+
+* [`gpii.firstDiscovery.attachTooltip.renderer`](attachTooltipRenderer.md)
+* [`fluid.prefs.panel`](http://docs.fluidproject.org/infusion/development/Panels.html)
+
+## Model
+
+This component supports the following
+[model](http://docs.fluidproject.org/infusion/development/tutorial-gettingStartedWithInfusion/ModelComponents.html)
+properties:
+
+| Path   | Description | Values | Default |
+|--------|-------------|--------|---------|
+| `lang` | The language applied to the First Discovery Tool | String | undefined |
+| `selectedLang` | The language code that its corresponding button has focus | String | undefined |
+| `viewportFirstLangIndex` | The index of the first visible language button | Number | 0 |
+| `atStartOfLangs` | Whether the first language button has focus | Boolean | false |
+| `atEndOfLangs` | Whether the last language button has focus | Boolean | false |
+
+## Options
+
+This component can be configured using the following
+[options](http://docs.fluidproject.org/infusion/development/ComponentOptionsAndDefaults.html):
+
+<table>
+    <tr><th>Name</th><th>Description</th><th>Values</th><th>Default</th></tr>
+    <tr>
+        <td>`numOfLangPerPage`</td>
+        <td>The number of language buttons to be displayed on the panel.</td>
+        <td>Number</td>
+        <td>
+        <pre><code>numOfLangPerPage: 3</code></pre>
```

Comment:
However, many other spots don't. For example the other table on this page and the next one - I think it would be less confusing if the "Default" column to contain nothing other than the actual default value